### ISSUE TITLE
Add basic zonal stats

### ISSUE THREAD

**Issue by lalligagger** [lalligagger/satgpt-app#5]
Would be really cool to add `rasterstats` tool to get e.g. state/ county-level statistics based on our indices or e.g. Land Surface Temp, once that's working.

**Comment by lalligagger** [comment]
Can load/ display geo vectors. "Just" need to add a new function/ tool for rasterstats. I will try this weekend and if I hit any issues I'm pushing this til after the contest.

**Comment by ivandorte** [comment]
I will try as well and let you know

**Comment by lalligagger** [comment]
This will need to be wrapped in a tool so check out my chatbot (chatbox) ui notebook for examples.

Here's how I used rasterstats for this most recently: https://gist.github.com/lalligagger/de430648b108d8f71f1e0f44942229be

**Comment by lalligagger** [comment]
Yes, I was thinking a couple of example GeoJSONs for the demo. Then users can add & point to their own.

Seattle has a nice neighborhood-level set of boundaries that would be fun with e.g. LST example.

**Comment by lalligagger** [comment]
I can try to tackle this one tonight.

**Comment by lalligagger** [comment]
Once we have all the other param/ plotting updates done this will be pretty straightforward. I can keep it on my plate for later in the week.
